internal/services/loadbalancer/loadbalancer_resource.go

```diff
@@ -121,6 +131,16 @@ func loadBalancer() *pluginsdk.Resource {
 							Set: pluginsdk.HashString,
 						},
 
+						"outbound_rule_ids": {
```

Comment:
could we ensure these are covered by a test?

Reply:
we can either open a new PR with that resource or add it to this one but that property needs to be be tested to be included here

Reply:
Removed the parameter because the resource `azurestack_public_ip_prefix` it's not supported at the current profile.